packages/schematics/angular/ai-config/index.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {
+  Rule,
+  apply,
+  applyTemplates,
+  chain,
+  filter,
+  mergeWith,
+  move,
+  strings,
+  url,
+} from '@angular-devkit/schematics';
+import { posix as path } from 'node:path';
+import { Schema as ConfigOptions, Tool } from './schema';
+
+const AI_TOOLS: { [key in Tool]: ContextFileInfo } = {
+  gemini: {
+    rulesName: 'GEMINI.md',
+    directory: '.gemini',
+  },
+  claude: {
+    rulesName: 'CLAUDE.md',
+    directory: '.claude',
+  },
+  copilot: {
+    rulesName: 'copilot-instructions.md',
+    directory: '.github',
+  },
+  windsurf: {
+    rulesName: 'guidelines.md',
+    directory: path.join('.windsurf', 'rules'),
+  },
+  jetbrains: {
+    rulesName: 'guidelines.md',
+    directory: '.junie',
+  },
+  // Cursor file has a front matter section.
+  cursor: {
+    rulesName: 'cursor.mdc',
+    directory: path.join('.cursor', 'rules'),
+    frontmatter: `---\ncontext: true\npriority: high\nscope: project\n---`,
+  },
+};
+
+interface ContextFileInfo {
+  rulesName: string;
+  directory: string;
+  frontmatter?: string;
+}
+
+export default function (options: ConfigOptions): Rule {
+  const selectedTools: Tool[] = options.tool;
+  const files: ContextFileInfo[] = selectedTools.map((selectedTool) => AI_TOOLS[selectedTool]);
```

Comment:
I think, if `none` is passed as a value, it will cause the schematic to error.

Probable this should exist early if `none` is passed and also maybe add a validation in the schema that does not allow `none` and some other value.


```
  "if": {
    "properties": {
      "aiConfig": {
        "contains": {
          "const": "none"
        }
      }
    },
    "required": [
      "aiConfig"
    ]
  },
  "then": {
    "properties": {
      "aiConfig": {
        "maxItems": 1
      }
    }
},
```

Would be good to add a test.